lib/commands/import.js

```diff
@@ -57,7 +57,7 @@ const serviceMapping = {
       }))
     const processFn = service.performStream.bind(service)
     const finishFn = () => log.info(service.summaryReport(options.from))
-    return [ options, 'prices.*', processFn, finishFn ]
+    return [ options, '*', processFn, finishFn ]
```

Comment:
I feel that we should add the `prices` identifier back...

Since the CLI can accept objects in several formats, if there is no identifier for each object, several methods will be call even when not intended and since it's possible that two different method might have similar JSON structure, updates will be triggered.

So might be too risky to do that